tests/components/recorder/test_v32_migration.py

```diff
@@ -221,7 +221,9 @@ def _get_events_index_names():
 
 
 async def test_migrate_can_resume_entity_id_post_migration(
```

Comment:
I see the schema gets updated WARNING:homeassistant.components.recorder.migration:Database is about to upgrade from schema version: 32 to: 43

All the runtime migrations are skipped because they are patched out.

Than it starts again, the run time migrations run, and the ix_states_entity_id_last_updated_ts index is removed after the FK removal for mysql & postgresql, and table rebuild for sqlite